akka-docs/src/main/paradox/scala/actors.md

```diff
@@ -905,6 +905,29 @@ Messages marked with `NotInfluenceReceiveTimeout` will not reset the timer. This
 `ReceiveTimeout` should be fired by external inactivity but not influenced by internal activity,
 e.g. scheduled tick messages.
 
+<a id="actors-timers"></a>
+
+## Timers, scheduled messages
```

Comment:
actors.md is getting quite long, perhaps we should move some things to their own file... not sure what though :).